terraform/stacks/unimelb/data_archive/fastq_archive.tf

```diff
@@ -174,7 +174,8 @@ data "aws_iam_policy_document" "fastq_archive" {
       identifiers = ["472057503814"]
     }
     actions = sort([
-      "s3:List*"
+      "s3:List*",
+      "s3:GetObjectTagging"
```

Comment:
Not strictly necessary for my use-case, but could be good to match the analysis bucket with other `GetObject*` access? https://github.com/umccr/infrastructure/blob/d887aa134c3ad817fd1d68c26a5340d63b4c9d95/terraform/stacks/unimelb/data_archive/analysis_archive.tf#L84-L86

Reply:
The idea is to not allow access to the actual data. It's supposed to be an archive after all. 

We may make some exceptions, but at this point I'd like to keep it at the strictly necessary permissions...